mds/authent/migrations/0002_post_add_index_accestoken_revoked_after.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.5 on 2019-11-30 21:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('authent', '0001_squashed_0007_post_make_application_user_scope_blankable'),
```

Comment:
It should be `0004` depending on `0003`.

Reply:
We are in the authent app :)